frontend/src/index.jsx

```diff
@@ -5,11 +5,16 @@ import LandingPage from './Pages/LandingPage/LandingPage.jsx';
 import LandingPageMobile from './Pages/LandingPage/LandingPageMobile.jsx';
 import FAQPage from './Pages/FAQPage/FAQPage.jsx';
 import AboutPage from './Pages/AboutPage/AboutPage.jsx';
+import PrizesPage from './Pages/Prizes/prizes.jsx';
```

Comment:
Why is there a new prizes page being added?

Reply:
I branched off the prizes branch by accident because I was helping the prizes page team a little and forgot to checkout main again haha